FAQ/FAQ-Deployment.md

```diff
@@ -79,6 +81,44 @@ sudo ln -s /mnt/usb/docker /var/lib/docker
 
 If you want to mount specific folder (like only Bitcoin node data directory), please browse `/var/lib/docker/volumes` to chose the different docker volumes.
 
+### How do I activate Tor on my BTCPay Server?
+
+That's really easy: just log in your instance with SSH, and enter the `root/btcpayserver-docker` directory as root. There, type the two following command lines:
+```
+root/btcpayserver-docker $ BTCPAYGEN_ADDITIONAL_FRAGMENTS="$BTCPAYGEN_ADDITIONAL_FRAGMENTS;opt-add-tor"
+root/btcpayserver-docker $ . btcpay-setup.sh -i
+```
+
+Then wait a few minutes for the server to restart, and you're done!
+
+If you used to have other options passed with BTCPAYGEN_ADDITIONAL_FRAGMENTS, remember to add those as well. For example, if you pruned the blockchain, you need to pass this line instead of the one above:
```

Comment:
no you don't need `BTCPAYGEN_ADDITIONAL_FRAGMENTS="$BTCPAYGEN_ADDITIONAL_FRAGMENTS;opt-add-tor"` already keep what you had before.

Reply:
Ah? First time I did that, and bitcoind got stuck, looking at the output it was asking to be restarted with a -reindex, so I thought it was maybe because I didn't pass the prune argument again, so I did just that and it was back on track. But maybe I was having another issue then...
Would it help if I turn off Tor and try to activate it again just to check if I can reproduce the error?

Reply:
Yeah, give it a try, maybe you did typo.